src/Bab/RabbitMq/Response.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Bab\RabbitMq;
+
+class Response
```

Comment:
As you want. :)
If you want to be able to deal with other provider that's fine. :) If you want to be coupled with Guzzle that's OK for me too. :)

It's really up to you.